yb-voyager/cmd/common.go

```diff
@@ -92,7 +92,7 @@ func updateFilePaths(source *srcdb.Source, exportDir string, tablesProgressMetad
 		for _, key := range sortedKeys {
 			tableName := tablesProgressMetadata[key].TableName
 			fullTableName := tableName.ForKey()
-			table := tableName.ForMinOutput()
```

Comment:
Instead of adding the schema name in the datafile for the single schema case, can't we fix the place where this name is being used to fetch the column name? 
I think this change can affect various other assumptions in code because we might be relying on this name to be unqualified for the single schema case.

Reply:
`ForMinOutput` function return the name qualified or not based on what is the default schema.
As per our implementation default schema for PG is:
1. if there are multiple and public is there then public
2. if single is there then that(irrespective of public/non-public)
3. Other no default schema